tasks/88_record-collection/record-collection.js

```diff
@@ -0,0 +1,31 @@
+var collection = {
+  2548: {
+    album: "Slippery When Wet",
+    artist: "Bon Jovi",
+    tracks: ["Let It Rock", "You Give Love a Bad Name"]
+  },
+  2468: {
+    album: "1999",
+    artist: "Prince",
+    tracks: ["1999", "Little Red Corvette"]
+  },
+  1245: {
+    artist: "Robert Palmer",
+    tracks: []
+  },
+  5439: {
+    album: "ABBA Gold"
+  }
+};
+
```

Comment:
What happened to this? And yet again, the test required function, and this is a constant.
```suggestion
// Keep a copy of the collection for tests
var collectionCopy = JSON.parse(JSON.stringify(collection));

// Only change code below this line
```